src/sharedComponents/Navbar.jsx

```diff
@@ -27,12 +27,19 @@ export default function Navbar() {
   const isUncontrast = Site.type === "uncontrast";
 
   const graphs = isUncontrast
-    ? Object.values(graphsHeaders).filter(
-        (x) => x.siteToDisplay === "uncontrast" || x.siteToDisplay === "both"
-      )
-    : Object.values(graphsHeaders).filter(
+    ? Object.values(graphsHeaders)
+        .filter(
+          (x) => x.siteToDisplay === "uncontrast" || x.siteToDisplay === "both"
+        )
+        .filter(
+          (item) => item.text !== "Distribution of Experiments Across Parameter"
```

Comment:
Better not use a free string as filter (prefer a constant) if it's only temporary than ok